src/client/components/Map/Map.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import '.../../../node_modules/leaflet/dist/leaflet.css';
+import L from 'leaflet';
```

Comment:
L is a very non descriptive name. As if there can only be one object that ever starts with that letter